config/initializers/delayed_job_config.rb

```diff
@@ -0,0 +1,6 @@
+Delayed::Worker.destroy_failed_jobs = false
```

Comment:
They are not! The defaults are as follows:

- `destroy_failed_jobs` default is true (we probably ultimately want to change that since the info we need should be in logs but I figured we might want to just keep the failures for now)
- `max_attempts` default is 25, which seems excessive for this work
- `read_ahead` default is 5, which now that I'm thinking about we may actually want to revert since it's unlikely we're going to have more than 5 enqueued jobs anyway
- `default_queue_name` default is to not use a named queue. this resulted in jobs from not-harvesting showing up in the harvest running jobs list, but setting a named queue default and limiting that display to this queue fixed it. I'm now realizing that we're probably going to want to have multiple named queues eventually, but it still seems good to have a name for the default queue.
- `delay_jobs` default is to always delay jobs. seems like we probably don't want that in test env (although so far they are always called with `perform_now` so it doesn't really matter)
- `raise_signal_exceptions` default is false, where setting it to `:term` allows SIGTERM signals to raise an exception, abort and unlock the job (which I think is probably preferable, especially while we're still in testing mode and may in fact want to abort a long-running job)

Reply:
yeah those all make sense. Thanks.